html/global_attributes.json

```
@@ -743,6 +743,45 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "ignores_find_in_page": {
+          "__compat": {
+            "description": "Element is ignored for the purposes of find-in-page.",
+            "spec_url": "https://html.spec.whatwg.org/multipage/interaction.html#dom-inert",
```

Better spec URL to the section that defines this behavior:

```suggestion
            "spec_url": "https://html.spec.whatwg.org/multipage/interaction.html#inert-subtrees",
```